webpack.config.js

```diff
@@ -109,68 +151,73 @@ var options = {
                 description: process.env.npm_package_description,
                 version: process.env.npm_package_version,
                 ...JSON.parse(content.toString()),
-                content_security_policy: content_security_policy,
               })
             );
           },
         },
       ],
-      {
-        logLevel: 'info',
-        copyUnmodified: true,
-      }
-    ),
-    new CopyWebpackPlugin(
-      [
+    }),
+    new CopyWebpackPlugin({
+      patterns: [
         {
-          from: 'src/pages/Content/content.styles.css',
+          from: 'src/pages/ContentScripts/content.styles.css',
           to: path.join(__dirname, 'build'),
           force: true,
         },
       ],
-      {
-        logLevel: 'info',
-        copyUnmodified: true,
-      }
-    ),
-    new CopyWebpackPlugin(
-      [
+    }),
+    new CopyWebpackPlugin({
+      patterns: [
         {
-          from: 'src/assets/img/',
+          from: 'src/assets/img/main.png',
           to: path.join(__dirname, 'build'),
           force: true,
         },
       ],
-      {
-        logLevel: 'info',
-        copyUnmodified: true,
-      }
-    ),
+    }),
     new HtmlWebpackPlugin({
       template: path.join(__dirname, 'src', 'pages', 'Options', 'index.html'),
       filename: 'options.html',
       chunks: ['options'],
+      cache: false,
     }),
     new HtmlWebpackPlugin({
       template: path.join(__dirname, 'src', 'pages', 'Popup', 'index.html'),
       filename: 'popup.html',
       chunks: ['popup'],
+      cache: false,
     }),
-    new WriteFilePlugin(),
-    new CrxWebpackPlugin({
-      keyFile: 'build.pem',
-      contentPath: 'build',
-      outputPath: 'release',
-      name: 'hypercrx',
```

Comment:
crx-webpack-plugin is a webpack plugin to pack an unpacked extension into a packed .crx extension. 
When do we need to pack extension into the form of a .crx file? It is when we release it. So this lines of config should not be shown here, but, they are ought to be run in the build script only. In development phase, this packing step costs time but is useless.

So remove from here.